validator/sawtooth_validator/execution/executor.py

```diff
@@ -336,7 +337,8 @@ def _fail_transaction(self, txn_signature,
                 error_message,
                 error_data)
 
-    def _send_and_process_result(self, content, connection_id, signature):
+    def _send_and_process_result(self, raw_content, connection_id):
```

Comment:
Renaming this parameter from `content` to `raw_content` implies that it is the content bytes, based on your other usages.  I would suggest naming it `process_request`, which is far more specific than `content` to begin with.

Reply:
Ok, I will rename raw_content to process_request. This comment applies to _execute method as well.